uqcsbot/text.py

```diff
@@ -33,17 +33,30 @@ async def encoding_autocomplete(
 class Text(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
-        self.zalgo_menu = app_commands.ContextMenu(
-            name="Zalgo",
-            callback=self.zalgo_context,
-        )
-        self.bot.tree.add_command(self.zalgo_menu)
 
-        self.mock_menu = app_commands.ContextMenu(
-            name="Mock",
-            callback=self.mock_context,
+        ##        self.zalgo_menu = app_commands.ContextMenu(
```

Comment:
I'm impartial, as I don't use the context menu much. Potentially, the chosen options should be decided or at least approved by committee.